package.json

```diff
@@ -20,7 +20,7 @@
     "create-react-component": "yo react-domain"
   },
   "dependencies": {
-    "@theforeman/vendor": "^0.1.1",
+    "@theforeman/vendor": "1.4.1-update-redux.0",
```

Comment:
This is a temporary version for testing/reviewing - please do not merge with this version